apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -333,6 +333,19 @@ def from_anndata(
                 ) as data:
                     _maybe_set(x, "data", data, use_relative_uri=use_relative_uri)
 
+                for key in anndata.layers.keys():
+                    with create_from_matrix(
+                        X_kind,
+                        _util.uri_joinpath(measurement_X_uri, key),
+                        anndata.X,
+                        platform_config,
+                        ingest_mode,
+                        context,
+                    ) as layer_data:
```

Comment:
It looks like all of these matrices are going to have the same data to me. Is this intended?